index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CC-Global-Summit-2015</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <!-- <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato:wght@100;400&display=swap"> -->
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- nav section -->
+    <header class="">
+      <div class="navigation">
+        <div class="nav-container">
+              <div class="logout">
+                <a href="#">  <i class="fa fa-facebook-f"  id="facebook"></i></a>
+                <a href="#"></a> <i class="fa fa-twitter" ></i></a>
+                <a href="#" class="list">Logout</a>
+                <a href="#" class="list">Mypage</a>
+                <a href="#" class="list">Korean</a>
+              </div>
+          </div>
+             <div class="navbar-container">
+                <div class="logo"><img src="Capstone/Header/logo.png" alt="simmitlogo"></div>
+                <div ><img src="capstone/Header/close_mobile_menu.png" alt="close-icon" id="close-icon"></div>
+                  <div class="nav-list">
+                    <a href="about.html" class="nav-menu">About</a>
+                    <a href="#"class="nav-menu">Program</a>
+                    <a href="#"class="nav-menu">Join</a>
+                    <a href="#"class="nav-menu">Sponsor</a>
+                    <a href="#"class="nav-menu">News</a>
+                    <a href="index.html"class="nav-menu" id="cc">CC Campaign</a>
+                 </div>
```

Comment:
- [ ] Make sure you have a home page link from the mobile version, Currently, the CC campaign works as coming back to the home page. You can add an option for the mobile version popup menu.